.jenkins/dsl/jobs.groovy

```diff
@@ -107,27 +109,35 @@ void setupDeployJob(String jobFolder, KogitoJobType jobType) {
             stringParam('IMAGE_TAG', '', 'Image tag to use to deploy images')
 
             // Test config if needed specifics. Else test default config will apply.
-            booleanParam('KOGITO_IMAGES_IN_OPENSHIFT_REGISTRY', false, 'Set to true if kogito images for tests are in internal Openshift registry.In this case, KOGITO_IMAGES_REGISTRY and KOGITO_IMAGES_NAMESPACE parameters will be ignored')
+            booleanParam('KOGITO_IMAGES_USE_OPENSHIFT_REGISTRY', false, 'Set to true if kogito images for tests are in internal Openshift registry.In this case, KOGITO_IMAGES_REGISTRY and KOGITO_IMAGES_NAMESPACE parameters will be ignored')
             stringParam('KOGITO_IMAGES_REGISTRY', "${CLOUD_IMAGE_REGISTRY}", 'Test images registry')
             stringParam('KOGITO_IMAGES_NAMESPACE', "${CLOUD_IMAGE_NAMESPACE}", 'Test images namespace')
             stringParam('KOGITO_IMAGES_NAME_SUFFIX', '', 'Test images name suffix')
             stringParam('KOGITO_IMAGES_TAG', '', 'Test images tag')
+
             stringParam('EXAMPLES_URI', '', 'Git uri to the kogito-examples repository to use for tests.')
             stringParam('EXAMPLES_REF', '', 'Git reference (branch/tag) to the kogito-examples repository to use for tests.')
-
-            // Release information
-            stringParam('PROJECT_VERSION', '', 'Optional if not RELEASE. If RELEASE, cannot be empty.')
+            stringParam('EXAMPLES_IMAGES_CACHE_MODE', 'never', 'Set the examples images\' cache mode for the BDD tests. Default it will always build the image.')
```

Comment:
Is it parameter used just in BDD test itself?
If so, shouldn't it have default value `always`? Or is the value supplied when job is started?

Reply:
As the examples images build is extracted, we may want to run the BDD tests with rebuild of images from a different examples uri/ref. So I would prefer to leave it as a parameter for standalone run.
But yeah, most of the time, we will use the cache images.
Just thinking that I will add the `tests.runtime-application-image-name-prefix=examples` in the default config for tests.

Reply:
but if you prefer the default to be `always`, I have no problem with that.

Reply:
ok

Reply:
thinking...

Reply:
fine for me to keep it as never as default, with setting always for PR checks/nightlies
that way it will keep consistent when it is used without example images

Reply:
yep: https://github.com/kiegroup/kogito-pipelines/pull/122/files#diff-0d9bfe5e25ba056e7749e322ef219937ad6f9c82ecda7c7359be9e93e14d59b3R236